tests/django_compat.rs

```diff
@@ -90,7 +90,7 @@ async fn it_matches_django_capture_behaviour() -> anyhow::Result<()> {
 
         let sink = MemorySink::default();
         let timesource = FixedTime { time: case.now };
-        let app = router(timesource, sink.clone());
+        let app = router(timesource, sink.clone(), false);
 
         let client = TestClient::new(app);
         let mut req = client.post(&case.path).body(raw_body);
```

Comment:
nit: tests will break due to `/e` not being exposed anymore, I'll patch it myself to rewrite the path